ci/docs/build.sh

```diff
@@ -16,40 +16,45 @@ export DOCS_WORKSPACE=$WORKSPACE/docs
 export NIGHTLY_VERSION=${NIGHTLY_OVERRIDE:=$(echo $BRANCH_VERSION | awk -F. '{ print $2 }')}
 export CUDA_REL=${CUDA_VERSION%.*}
 export CUDA_SHORT=${CUDA_REL//./}
-export CLX_HOME=$WORKSPACE/clx_build
 export PROJECTS=(clx)
 
 # Switch to project root; also root of repo checkout
-cd $WORKSPACE
+cd $PROJECT_WORKSPACE
 export GIT_DESCRIBE_TAG=`git describe --tags`
 export MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
 
-logger "Check environment..."
+gpuci_logger "Check environment"
 env
 
-logger "Check GPU usage..."
+gpuci_logger "Check GPU usage"
 nvidia-smi
 
-logger "Activate conda env..."
-source activate rapids
-conda install --freeze-installed -c rapidsai-nightly -c rapidsai -c nvidia -c pytorch -c conda-forge \
+gpuci_logger "Activate conda env"
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
+
+gpuci_conda_retry install --freeze-installed -c rapidsai-nightly -c rapidsai -c nvidia -c pytorch -c conda-forge \
     pytorch torchvision requests yaml python-confluent-kafka python-whois markdown beautifulsoup4 jq
```

Comment:
```suggestion
    pytorch torchvision requests yaml python-confluent-kafka python-whois
```

A few of these can be removed.